docs/website/data/integrations.yaml

```diff
@@ -385,6 +385,18 @@ integrations:
     videos:
       - https://www.youtube.com/watch?v=9m4FymEvOqM&feature=share
 
+  clojure:
+    title: App authorization for Clojure
```

Comment:
Just how common is Ring? Worth mentioning in the title?

Reply:
It's pretty much _the_ HTTP server abstraction for Clojure, but I think having it in the description is enough since most probably won't know what it is, and the library could grow to include more use cases later.